README.md

```diff
@@ -32,9 +32,12 @@ Furthermore *EMBArk* improves the data provision by aggregating the various scan
 *EMBArk* is using the firmware security scanner *EMBA* as backend. You can take a look at the [*EMBA*](https://github.com/e-m-b-a/emba) repository and get more [information about usage of *EMBA* in the wiki](https://github.com/e-m-b-a/emba/wiki/Usage). Additionally you should check the [*EMBArk* wiki](https://github.com/e-m-b-a/embark/wiki).
 
 ## Start Server
-To start the EMBArk-Server simply run `$ sudo ./embark/run-server.sh`.
+To start the EMBArk-Server simply run `$ sudo pipenv run ./run-server.sh`.
```

Comment:
in the installation script you are refering to
`Ready to use $sudo ./run-server.sh `